website/docs/guides/core-to-cloud-3.md

```diff
@@ -0,0 +1,188 @@
+
+The dbt Cloud IDE provides a simplified interface that's accessible to all users, regardless of their technical background. However, there are some capabilities that are intentionally not available in the dbt Cloud IDE due to its focus on simplicity and audience:
+
+- Pre-commit for automated checks before *committing* code is not available (yet).
+- Mass-generating files / interacting with the file system are not available.
+- Combining/piping commands, such as `dbt run -s (bash command)` is not available.
+
+### dbt Cloud CLI
+The dbt Cloud CLI allows you to run dbt commands against your dbt Cloud development environment from your local command line. For users who seek full control over their development environment and ideal for those comfortable with the command line.
+
+**Who might prefer the dbt Cloud CLI?**
+
+- Experienced dbt users and developers looking for granular control over their Git workflows.
+- Data engineers and practitioners who need to perform complex operations, like mass file generation or specific command combinations.
+
+**Key features**
+
+- Allows users to run dbt commands against their dbt Cloud development environment from their local command line with minimal configuration.
+- Allows users to natively [defer to production](/docs/cloud/about-cloud-develop-defer#defer-in-dbt-cloud-cli) metadata directly in their development workflows, reducing the number of objects.
+- Support running multiple dbt commands at the same time through [safe parallel execution](/reference/dbt-commands#parallel-execution), a [feature](/docs/cloud/about-cloud/dbt-cloud-features) available in dbt Cloud's infrastructure. In contrast, `dbt-core` *doesn't support* safe parallel execution for multiple invocations in the same process.
+- Able to use Visual Studio (VS) Code extensions (such as [dbt-power-user](https://marketplace.visualstudio.com/items?itemName=innoverio.vscode-dbt-power-user) to enhance the development experience by adding extra functionalities.
+
+## Orchestration
+
+dbt Cloud provides a robust orchestration that enables users to schedule, run, and monitor dbt jobs with ease. Here are some tips and caveats to consider when using dbt Cloud's orchestration features:
+
+### Tips
+
+- Enable [partial parsing](/docs/deploy/deploy-environments#partial-parsing) between jobs in dbt Cloud to significantly speed up project parsing by only processing changed files, optimizing performance for large projects.
+- [Run multiple CI/CD](/docs/deploy/continuous-integration) jobs at the same time which will not block production runs. The Job scheduler automatically cancels stale runs  when a newer commit is pushed. This is because each PR will run in its own schema.
+- dbt Cloud automatically cancels a scheduled run if the existing run is still executing. This prevents unnecessary, duplicative executions.
```

Comment:
This is more of a notification than a tip. Since it's automatic, does the user need to do any configuration or can we suggest an action for them to make this even more effective?

Reply:
they don't but i thought it'd be helpful to know how it works in cloud. they do need to know that api jobs aren't covered so I'll link to the section.